OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/CourseOutlineAdapter.java

```diff
@@ -69,8 +73,14 @@
     private Config config;
     private boolean isVideoMode;
 
+
+    private String lastAccessedUnitId;
+    private boolean hasLastAccessedUnitShown = false;
+    private int lastAccessedUnitPosition = -1;
+    public HashSet<Integer> selectedItemPositions;
```

Comment:
⚠️ 
As per discussion, we don't need these class variable here for managing the last access item themes.
We can simply introduce index mechanism which we shall calculate at initial time only.

Reply:
As per dicussion:
We can also remove `lastAccessedUnitId` from here, specially there is no need to pass it on constructor.
Also check if we can remove `selectedItemPosition` by analyzing the already multi-selection functionality.